microprofile/telemetry/pom.xml

```diff
@@ -137,12 +137,15 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <forkCount>1</forkCount>
-                    <reuseForks>false</reuseForks>
-                    <!-- The default for the following is 5000 ms. Reducing it significantly speeds up tests. -->
-                    <argLine>-Dotel.bsp.schedule.delay=100</argLine>
-                </configuration>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <!-- The default for the following is 5000 ms. Reducing it significantly speeds up tests. -->
+                            <argLine>-Dotel.bsp.schedule.delay=100</argLine>
```

Comment:
Consider, rather:
```
<systemPropertyVariables>
  <otel.bsp.schedule.delay>${otel.bsp.schedule.delay}</otel.bsp.schedule.delay>
</systemPropertyVariables>
```
…and, elsewhere in the `pom.xml`:
```
<properties>
  <otel.bsp.schedule.delay>100</otel.bsp.schedule.delay>
</properties>
```

Reply:
Fixed